framework-core/src/main/java/org/daisy/pipeline/properties/Properties.java

```diff
@@ -0,0 +1,24 @@
+package org.daisy.pipeline.properties;
+
+import java.util.Map;
+
+public class Properties {
```

Comment:
if this class isn't intended to be extended, better make it `final` already, and if it's only intended as a static helper, we can add a private constructor too.

Reply:
OK good point. I normally prefer`final abstract`, is that OK too?

Reply:
OK.

Reply:
`final abstract` is not allowed. Normally I just add `abstract` without `final`. So your suggestion (`final` + private contructor) is the only real solution.